Templating/VOKTemplateModel.m

```diff
@@ -25,7 +25,7 @@ @interface VOKTemplateModel ()
 static NSString *const KitNameIOS = @"UIKit";
 static NSString *const KitNameMac = @"AppKit";
 
-static NSString *const FrameworkPrefix = @"ac_";
+static NSString *const MethodNamePrefix = @"ac";
```

Comment:
Would it make sense to leave the `_` as part of the prefix, so that a user could specify a prefix without the `_` separating the prefix from the method name?